monitoring/podMonitor.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: dragonfly-monitor
```

Comment:
because these pod monitors are specific to the `dragonfly-sample` DF instance, I'd advice on having the same name i.e `dragonfly-sample` instead of generic naming